src/backend/marsha/core/views.py

```diff
@@ -117,6 +127,17 @@ def model(self):
     def serializer_class(self):
         """Return the serializer used by the view."""
 
+    def _get_base_app_data(self):
+        """Define common app data for the LTI view."""
+        app_data = _get_base_app_data()
+        app_data.update(
+            {
+                "frontend": "LTI",
```

Comment:
This is already defined in _get_base_app_data, do you think we should leave it here again ?

Reply:
Yes, I have to remove it in the `_get_base_app_data` function.